sdk/eventhub/ci.yml

```diff
@@ -0,0 +1,18 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+trigger: none
+
+pr:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - sdk/eventhubs/
+
+variables:
+  build_targeting_string: 'azure-eventhubs'
```

Comment:
This is `BuildTargetingString` now instead of `build_targeting_string` [Master](https://github.com/Azure/azure-sdk-for-python/blob/779606f0c3ee2666f5a8ef3f809a0f15dbfc0861/.azure-pipelines/templates/steps/archetype-sdk-client.yml#L30)

Reply:
In ```/eng/pipelines/templates/jobs/archetype-sdk-client.yml``` it is still ```build_targetting_string```. At the moment I've got a copy of what was in ```client.yml``` I think we should now collapse these two together by changing ```client.yml`` to reference ```pipelines/templates/jobs/achetype-sdk-client.yml``` as a template so that everything is consistent.

We can then change ```archetype-sdk-client.yml``` to use ```BuildTargettingString``` as the variable name - make sense?

Reply:
Ok.